persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/BotServiceAdapter.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.kalium.persistence.dao
+
+import app.cash.sqldelight.ColumnAdapter
+
+class BotServiceAdapter : ColumnAdapter<BotEntity, String> {
```

Comment:
Doesn't this fall into the same category of qualified IDs ? Just asking to see if there were some re-utilization case or not 😅 ?

Which values are possibles in the `provider` field ?

Reply:
Currently we don't have provider name which probably will be added to bot service in future 🤔